src/features/TakeNow/ui/MultiInformantTooltip.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import { Box } from '@mui/material';
+import Avatar from '@mui/material/Avatar';
+import Tooltip from '@mui/material/Tooltip';
+
+import TooltipIcon from '~/assets/tooltip-icon-i.svg';
+import { useMultiInformantState } from '~/entities/applet/model/hooks';
+import { MultiInformantTooltipText } from '~/features/TakeNow/ui/MultiInformantTooltipText';
+import { Theme } from '~/shared/constants';
+import { useCustomTranslation } from '~/shared/utils';
+import { useFeatureFlags } from '~/shared/utils/hooks/useFeatureFlags';
+
+export const MultiInformantTooltip = () => {
+  const { t } = useCustomTranslation();
+  const { getMultiInformantState, isInMultiInformantFlow } = useMultiInformantState();
+  const { featureFlags } = useFeatureFlags();
+  if (!isInMultiInformantFlow() || !featureFlags.enableMultiInformant) {
+    return null;
+  }
+
+  const { sourceSubject, targetSubject, currentUserSubject } = getMultiInformantState();
+
+  if (!sourceSubject || !targetSubject || !currentUserSubject) {
+    return null;
+  }
+
+  return (
+    <Tooltip
+      title={
+        <Box
+          padding="4px 8px"
+          maxWidth="none"
+          borderRadius="4px"
+          sx={{ backgroundColor: Theme.colors.light.inverseSurface }}
+        >
+          <MultiInformantTooltipText
+            caption={t('takeNow.tooltip.providingAnswers')}
+            subject={sourceSubject}
+          />
+          <MultiInformantTooltipText
+            caption={t('takeNow.tooltip.inputtingAnswers')}
+            subject={currentUserSubject}
+          />
+          <MultiInformantTooltipText
+            caption={t('takeNow.tooltip.subjectOfAnswers')}
```

Comment:
Looks like design has [updated the language](https://www.figma.com/design/6BNx63peNVzrHAF1FoZ78x/Multi-Informant-2024?node-id=6026-303132&t=pntZXC2cWLgB76WL-4) since the demo:

```suggestion
            caption={t('takeNow.tooltip.providingResponses')}
            subject={sourceSubject}
          />
          <MultiInformantTooltipText
            caption={t('takeNow.tooltip.inputtingResponses')}
            subject={currentUserSubject}
          />
          <MultiInformantTooltipText
            caption={t('takeNow.tooltip.subjectOfResponses')}
```